src/server/utilities/podcastExternalLinkData/index.js

```diff
@@ -0,0 +1,34 @@
+/*
+  Do not import this file into the primary application.
+  This file is intended for use by tests only where needed.
```

Comment:
Is `server/utilities` the best place for this?  maybe it should be alongside the tests that use it, or inside `src/testHelpers` if it's going to be used in many places